verification.py

```diff
@@ -363,6 +363,58 @@ def f():
             self._run_mutation_discoal_stats(key, command_line)
         self._instances[key] = f
 
+    def run_sweep_comparison(self, key, args):
+        """
+        does a comparison of sweep model vs discoal
+        """
+        df = pd.DataFrame()
+        refsize = 1e4
+        nreps = 1000
+        mod = msprime.SweepGenicSelection(refsize)
+        mod.start_frequency = 1.0 / (2 * refsize)
+        mod.end_frequency = 1.0 - (1.0 / (2 * refsize))
+        mod.alpha = 1000.0
+        mod.dt = 1.0/(40 * refsize)
+        seqlen = 1e4
+        mu = 2.5e-4
+        rho = 2.5e-4
+        mod.position = np.floor(seqlen/2)
+        data = collections.defaultdict(list)
+        replicates = msprime.simulate(10,  model=mod, length=seqlen,
+                                      num_labels=2, recombination_rate=rho,
+                                      num_replicates=nreps)
+        for x in replicates:
+            ts = msprime.simulate(from_ts=x, recombination_rate=rho)
```

Comment:
this is a bit ugly, but I'm forced to do this as our current implementation of the `SweepGenicSelection()` model doesn't finish all the coalescing for us. so what i've done here is start a second round of simulation off from the tree sequence that the sweep simulation ends with. 

@jeromekelleher what do you think here?

Reply:
How about adding  a ``demographic_events=SimulationModelChange(time=None, model="hudson")`` ? That was the idea anyway, I'm not sure if I got around to implementing this. We should change the default value for ``time`` to None if this works, so we can just say

```
ts = msprime.simulate(
      10,  
      model=selection_model, 
      demographic_events=[SimulationModelChange(model="hudson")])
```
and it'll do what we want.

Reply:
Yeah it doesn't look like a `time=None` option has been implemented for this situation where you want it to change models at some arbitrary time.

Reply:
seems to me like setting `time=np.Inf` is doing the correct thing but i'm not 100% certain